platform/ios/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## master
+## 3.7.0
```

Comment:
After looking side to side the changelog I'm not sure if we should let 3.7.0 be the header. What do you think @boundsj  @1ec5?

Reply:
This happens whenever we wait too long to merge back to master. The header should say “3.7.0”, but we’ll need to manually move some of the entries up into their own “master” section.